scenarios/modules/loadbalancer/settings.tf

```diff
@@ -0,0 +1,8 @@
+terraform {
```

Comment:
Modules should not be used alone, so  `settings.tf` is redundant

Reply:
nope, it should be here, if we remove this file, it will try to download provider from `hashicorp/opentelekomcloud`